### ISSUE TITLE
Lines starting with obsidian tags gets parsed as a header

### ISSUE THREAD

**Issue by cramt** [Praxxian/lava-flow#34]
Obsidian:
![image](https://user-images.githubusercontent.com/23054512/221406011-62c7c13e-f3b1-4fd5-9464-524093aa24ee.png)
Foundry:
![image](https://user-images.githubusercontent.com/23054512/221405997-7e8cb159-25ce-4260-b052-7d8e7b764b8b.png)

The difference between a tag and a header can be known by whether or not there is a space between the `#` and the next eg `# header`, `#tag`

**Comment by Praxxian** [comment]
Looks like showdown, the library used to convert markup, has an option for this that is off by default (requireSpaceBeforeHeadingText). I'll see if I can set this flag.

**Comment by Praxxian** [comment]
Fixed in release 3.0.2.

**Comment by denver-andy** [comment]
This issue does not appear to be resolved. 

I installed lava-flow today (v3.1.0 on Foundry v11 build 315). When I imported my vault, my tags are showing as headers. 

To reproduce:

Markdown file "Dripping Blood Tribe.md":
`#Session000 #Faction

A tribe of goblins that is growing numbers. Active in the [[Central Valley]], it is believed that a powerful shaman, versed in blood magic and dark rituals, leads the tribe. Rumors say that the shaman can summon creatures to fight for the goblins and to cast curses on their foes.`

Results in:
<img width="834" alt="Screenshot 2024-04-05 at 4 09 26 PM" src="https://github.com/Praxxian/lava-flow/assets/67066507/1aa3c155-fe73-4513-8090-8146d6f6cc3f">
